scripts/pyomo_uninstall.py

```diff
@@ -39,26 +38,6 @@
 'coopr.pysp',
 'coopr.solvers',
 'coopr.sucasa',
-'pyutilib.R',
-'pyutilib.autotest',
-'pyutilib.common',
-'pyutilib.component.app',
-'pyutilib.component.config',
-'pyutilib.component.core',
-'pyutilib.component.executables',
-'pyutilib.component.loader',
-'pyutilib.dev',
-'pyutilib.excel',
-'pyutilib.math',
-'pyutilib.misc',
-'pyutilib.ply',
-'pyutilib.pyro',
-'pyutilib.services',
-'pyutilib.subprocess',
-'pyutilib.svn',
-'pyutilib.th',
-'pyutilib.virtualenv',
-'pyutilib.workflow',
```

Comment:
I would ether leave this script untouched, or else remove the entire thing.  It was really only needed back in the Coopr days - and that has been years.

Reply:
I'm all for removing.